python/cugraph/cugraph/dask/sampling/neighborhood_sampling.py

```diff
@@ -0,0 +1,165 @@
+                         with_replacement=True):
+    """
+    Does neighborhood sampling.
+
+    Parameters
+    ----------
+    input_graph : cugraph.DiGraph
+        cuGraph graph, which contains connectivity information as dask cudf
+        edge list dataframe
+
+    start_info_list : list
+        ...
+
+    fanout_vals : list
+        List of branching out (fan-out) degrees per starting vertex for each
+        hop level
+
+    with_replacement: bool, optional (default=True)
+        Flag to specify if the random sampling is done with replacement
+
+    Returns
+    -------
+    result : dask_cudf.DataFrame
+        GPU data frame containing two dask_cudf.Series
+
+        ddf['srcs']: dask_cudf.Series
+            Contains the source vertices from the sampling result
+        ddf['dsts']: dask_cudf.Series
+            Contains the destination vertices from the sampling result
+        ddf['labels']: dask_cudf.Series
+            Contains the start labels from the sampling result
+        ddf['index']: dask_cudf.Series
+            Contains the indices from the sampling result
+        ddf['counts']: dask_cudf.Series
+            Contains the transaction counts from the sampling result
+    """
+
+    print("Hello from cugraph/dask!")
+
+    # Initialize dask client
+    client = default_client()
+    # Important for handling renumbering
+    input_graph.compute_renumber_edge_list(transposed=True)
+
+    ddf = input_graph.edgelist.edgelist_df
+    # vertex_partition_offsets = get_vertex_partition_offsets(input_graph)
+    # num_verts = vertex_partition_offsets.iloc[-1]
+    num_edges = len(ddf)
+    data = get_distributed_data(ddf)
+
+    src_col_name = input_graph.renumber_map.renumbered_src_col_name
+    dst_col_name = input_graph.renumber_map.renumbered_dst_col_name
+
+    """
+    # Would want G or whatever takes its place to be a pylibcugraph MG Graph,
+    # which isn't implemented yet but will be made. The pylib MG Graph will use
+    # cugraph_mg_graph_create, which means that #2110 is a dependency
+    return pylibcugraph.uniform_neighborhood_sampling(G, start_info_list,
+                                                      fanout_vals,
+                                                      with_replacement)
+    """
+    result = [client.submit(call_nbr_sampling,
+                            Comms.get_session_id(),
+                            wf[1],
+                            src_col_name,
+                            dst_col_name,
+                            num_edges,
+                            False,
+                            start_info_list,
+                            fanout_vals,
+                            with_replacement,
+                            workers=[wf[0]])
+              for idx, wf in enumerate(data.worker_to_parts.items())]
+
+
+    wait(result)
+
+    cudf_result = [client.submit(convert_to_cudf,
+                                 cp_arrays)
+                   for cp_arrays in cupy_result]
```

Comment:
cupy_result is not reference anywhere. I believe you forgot to update the variable `result` to `cupy_result`

Reply:
Thanks for catching that.